app.py

```diff
@@ -1,5 +1,6 @@
 from flask import Flask, render_template, jsonify
 from flask_apscheduler import APScheduler
+import psutil
```

Comment:
You should then remove the import of a thing which is not used, isn't it?

Reply:
yes sir it is

Reply:
I am done updating the imports in the app.py